__tests__/api.test.js

```diff
@@ -96,22 +97,25 @@ describe("GET /api/articles", () => {
 });
 
 describe("GET /api/articles/:article_id/comments", () => {
-  test("GET:200 gets comnments of a specific article", () => {
+  test("GET:200 gets comments of a specific article", () => {
     return request(app)
-      .get("/api/articles/2/comments")
+      .get("/api/articles/1/comments")
       .expect(200)
       .then((res) => {
-        articles = [res.body.article];
-        articles.forEach((article) => {
-          expect(article).toHaveProperty("comment_id");
-          expect(article).toHaveProperty("votes");
-          expect(article).toHaveProperty("created_at");
-          expect(article).toHaveProperty("author");
-          expect(article).toHaveProperty("body");
-          expect(article).toHaveProperty("article_id");
+        const comments = res.body.article;
+        console.log(typeof comments);
+        comments.forEach((comment) => {
+          console.log(comment);
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
         });
       });
   });
+
```

Comment:
You will need a test for :
- when an article exists but has no comments
- when the article_id is invalid 
- when the article_id is valid but non existing